lib/improver/cli/estimate_emos_coefficients.py

```diff
@@ -133,6 +187,29 @@ def process(historic_forecast, truth, distribution, cycletime, units=None,
             The cube containing the historical forecasts used for calibration.
         truth (iris.cube.Cube):
             The cube containing the truth used for calibration.
+        combined (iris.cube.CubeList):
+            A cubelist containing a combination of historic forecasts and
+            associated truths.
+        historic_forecast_dict (dict):
+            Dictionary specifying the metadata that defines the historic
```

Comment:
Again maybe we could link to the plugin that uses this dictionary. What do you think?

Reply:
I've added a link to the CLI help.